azure/arm/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure Resource Manager template
+
+This ARM template installs the Datadog Agentless Scanner.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FDataDog%2Fterraform-module-datadog-agentless-scanner%2Frefs%2Fheads%2Fmain%2Fazure%2Farm%2Fmain.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FDataDog%2Fterraform-module-datadog-agentless-scanner%2Frefs%2Fheads%2Fmain%2Fazure%2Farm%2FuiFormDefinition.json)
```

Comment:
In case you want to test the version in this branch:

[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FDataDog%2Fterraform-module-datadog-agentless-scanner%2Frefs%2Fheads%2Fdiogocp%2FSEC-16914-azure-bicep%2Fazure%2Farm%2Fmain.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FDataDog%2Fterraform-module-datadog-agentless-scanner%2Frefs%2Fheads%2Fdiogocp%2FSEC-16914-azure-bicep%2Fazure%2Farm%2FuiFormDefinition.json)